src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
```

Comment:
yeah, but all this is still scoped to social, right?  an alloy site wouldn't need this without social.